games-fps/eduke32/eduke32-20180224_6694.recipe

```diff
@@ -31,13 +31,14 @@ years ago
 * EDuke32 makes sandwiches!"
 HOMEPAGE="http://www.eduke32.com/"
 COPYRIGHT="1993-1997 Ken Silverman
-	2006-2017 eduke32 Team"
-LICENSE="GNU GPL v2"
-REVISION="3"
-SOURCE_URI="http://dukeworld.duke4.net/eduke32/synthesis/20170518-6135/eduke32_src_20170518-6135.tar.xz"
-CHECKSUM_SHA256="539f928f9cee88a3d1dbcbf1f99a2e44e2d078185f99cf4ab6ed05922d450d2d"
-SOURCE_DIR="eduke32_20170518-6135"
-PATCHES="eduke32-20170518_6135.patchset"
+	2006-2018 eduke32 Team"
+LICENSE="GNU GPL v2
+	BUILD"
```

Comment:
We cannot use a name such as `BUILD` for a license.
You should either:
* rename that file to something more specific, maybe `Ken Silverman Build Source Code License` (or anything else but `BUILD`) and adjust `LICENSE` accordingly,
* or remove `BUILD` from `LICENSE` and modify `INSTALL()` to have this license copied to `$docDir`:
```diff
 INSTALL()
 {
 	mkdir -p $binDir
 	cp eduke32 mapster32 $binDir
 	strip $binDir/*
+	mkdir -p "$docDir"
+	cp package/common/buildlic.txt "$docDir"
 }
```

Reply:
Why not? It's the name that the author selects.

Reply:
Because the name `BUILD` is too generic.
Having such a name for a file in `/system/data/licenses/` would be a very bad thing. FIles in that directory should have meaningful names or names that can be easily associated to the package they belong to.

Reply:
I disagree.

The name `BUILD` is no less meaningful than `SleepyCat` or `Digital Mars` (used by sys-db/db and dev-lang/dmd), so this is purely subjective. 

I do agree however, it's quite hard to figure out which package uses this license, as I can't seem to figure out any options to make `pkgman` spits that information out, so here is my Google-friendly name proposal: `Build Engine`

Reply:
I have to admit that "meaningful" was not the right word.

Anyway, I like your proposal for `Build Engine` :)

I don't think there is an easy way to list all the packages which have a given license.
That said, you can try:
```
listattr -l /path/to/file | grep -w PACKAGE
```
and it will give the name of the one of the packages that has that file.

And if you want to have all the packages that use the MIT license:
```
( for i in /system/packages/*.hpkg; do package list -i $i | grep license: | sed "s|^|$i|" ; done ) | grep MIT
```

But maybe there is a better way to query the filesystem.